code/modules/client/preference_setup/loadout/loadout_head.dm

```diff
@@ -4,10 +4,6 @@
 	slot = slot_head
 	sort_category = "Hats and Headwear"
 
-/datum/gear/head/ushanka_grey
-	display_name = "ushanka, grey"
-	path = /obj/item/clothing/head/ushanka/grey
-
```

Comment:
Should be kept. Unique in it's own right.

Example : Brown and Black leather jackets.